doc/usage.md

```diff
@@ -573,14 +573,19 @@ error result.
 
 ## How to Use Compliance Operator with HyperShift Management Cluster
 
-Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
-for the Hosted Cluster with a tailoredProfile.
+[Hypershift](https://hypershift-docs.netlify.app/) allows one to create and manage clusters on existing infrastructure.
+Compliance Operator is able to create a platform scan on the [HyperShift Management Cluster](https://hypershift-docs.netlify.app/reference/concepts-and-personas/)
+for the Hosted Cluster with a TailoredProfile.
 
-Currently, we only support CIS profile and PCI-DSS profile, in order to scan Hosted
-Cluster, you need to create a tailoredProfile and then set the value of
-`ocp4-hypershift-cluster` to the name of the Hosted Cluster you want to scan,
+Currently, we only support CIS profile and PCI-DSS profile,
 you can either extend `ocp4-cis` or `ocp4-pci-dss`.
 
+In order to scan a Hosted Cluster, you need to create a TailoredProfile specifying the
+name and namespace of the Hosted cluster that you want to scan.
```

Comment:
nit: Hosted Cluster (to be consistent with the line above).

Reply:
Addressed.